documentation/docs/configuration.md

```diff
@@ -315,6 +318,38 @@ if [`get5_print_damage`](#get5_print_damage) is disabled.<br>
 ####`get5_knife_cfg`
 :   Config file executed for the knife round, relative to `csgo/cfg`.<br>**`Default: "get5/knife.cfg"`**
 
+####`get5_autoload_config`
```

Comment:
Remove `get5_autoload_config` - it's up under server config now. You have it in here twice when adding it here.